components/layout_2020/flexbox/layout.rs

```diff
@@ -328,29 +328,38 @@ impl FlexContainer {
                             free_space
                         }
                     },
-                    AlignContent::Center => free_space / 2.0,
+                    AlignContent::Center => {
+                        if free_space >= Length::zero() {
+                            free_space / 2.0
+                        } else {
+                            Length::zero()
+                            // TODO: implement unsafe fallback: (free_space / 2.0)
+                        }
+                    },
```

Comment:
`center` should default to unsafe, it was previously fine.

https://drafts.csswg.org/css-align/#typedef-overflow-position

> UAs that have not implemented the “smart” default behavior must behave as [safe](https://drafts.csswg.org/css-align/#valdef-overflow-position-safe) for [align-content](https://drafts.csswg.org/css-align/#propdef-align-content) on [block containers](https://drafts.csswg.org/css-display-4/#block-container) and [unsafe](https://drafts.csswg.org/css-align/#valdef-overflow-position-unsafe) otherwise.

Reply:
Hmm... the spec for fallbacks (https://drafts.csswg.org/css-align/#distribution-values) seems a little inconsistent:

- `space-between` and `stretch` fallback to `flex-start`
- `space-around` and `space-evenly` fallback to `safe center` which falls back `start` (not `flex-start`)

So `space-between`/`stretch` are going to align to the opposite end of the container to `space-around`/`space-evenly` in the "single alignment subject overflows container" case.

I feel like either `space-between` and `stretch` ought to fallback to `start` or `space-around` and `space-evenly` ought to fallback to `flex-start` and the behaviour of "distributed alignment" keywords in the "single alignment subject" case ought to be defined via a mechanism other than fallback.

Reply:
Good point. Seems that Blink makes them fall back to flex-start.